### PR TITLE
fix(website): #244 sort events

### PR DESCRIPTION
**Pull request by Han2-Ro**
Closes: #244

### CODE REVIEW

File: src/data/events.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export async function getNext3Events(): Promise<
   const today = new Date();
   today.setHours(0, 0, 0, 0);
   const request = {
-    sort: 'date',
+    sort: 'start',
```
Comment (Han2-Ro) [inline src/data/events.ts:145]: So this was a surprisingly easy fix :) . I think it's still from the time we had only one date and not a start and a end date.

Comment (MrGreenTea) [reply src/data/events.ts:145]: @Han2-Ro oh right. I wonder if we can somehow make sure this doesn't happen again? For example by using the payload types.

**Comment by MrGreenTea** [comment]
@Han2-Ro The commit message could use some better detail (something like "Fix sorting"). In general the commit message should be more about the "Why"/Reasons of the change. The "What" can be inferred from the code changes / diff :)

**Review by MrGreenTea — APPROVED** [review]

File: src/data/events.ts (hunk 1 above)
Comment (Han2-Ro) [reply src/data/events.ts:145]: yeah but I have no idea how you would take the name of a property (here: event.start) and use it as the value of a string. Or is there some other way?

Comment (MrGreenTea) [reply src/data/events.ts:145]: I think we should think about this another time as it's ok right now. I have some ideas we can talk about tomorrow like building a small client abstraction.

**Pull request merged**